scripts/setup.sh

```diff
@@ -5,33 +5,41 @@
 # virtual TDP cluster using the TDP-getting-started repo
 ###
 
-# Get tdp-getting-started root dir
-root_dir="$(dirname "$0")/.."
+# tdp-collection
+TDP_COLLECTION_URL=https://github.com/TOSIT-IO/tdp-collection
+TDP_ROLES_PATH=ansible_roles/collections/ansible_collections/tosit/tdp
+TDP_COLLECTION_COMMIT=86f2d3f42df18a5ac07cc25e847ddaf3082be6d4
```

Comment:
My opinion is that the person contributing a breaking change to `collection` or `collection-extra` should be in charge of making the change to fix at `getting-started` at the same time. If said contributor is not usually active on `getting-started`, it is unlikely that no one will notice the breaking change PR since we are all actors of the `collection` or `collection-extra` repos

Reply:
The reason I'm forcing a commit here is to avoid that future updates stop getting started from being deployable due to changes in tdp-collection and tdp-collection-extras.

I don't think we can avoid locking the other collection versions which are downloaded because the getting-started repo is a sandbox which can easily be broken by external collection updates. If the getting-started repo were being used as an automated daily integration test across all repos, I'd agree to always pull the latest, but its current purpose is a sandbox so I think that it has to be stable and reliable.

I've no objection to using tags instead but I think we should merge this PR as a temporary measure to keep the getting-started sandbox stable until the tagging conventions are decided and tags are in use (then we update to tag versioning).

Reply:
> My opinion is that the person contributing a breaking change to `collection` or `collection-extra` should be in charge of making the change to fix at `getting-started` at the same time. If said contributor is not usually active on `getting-started`, it is unlikely that no one will notice the breaking change PR since we are all actors of the `collection` or `collection-extra` repos

I agree but in deploying the getting-started from scratch is the only test which catches all issues and its not something we can enforce as far as I know.

It would be a nice idea to have a daily jenkins task to redeploy from scratch on the latest masters to have some warning about when its failing.

Reply:
Ok let's do like this for now temporarily and keep in mind we should change this once we have tags.

@Nuttymoon mentionned a `dev-setup.sh` we could also use to have some point.